js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx

```diff
@@ -44,6 +45,9 @@ interface RunsFeedTableProps {
   scroll?: boolean;
 }
 
+// Potentially other modals in the future
+type RunsFeedModalState = {type: 'partitions'; backfillId: string};
```

Comment:
Nit: can we use "dialog" instead of "modal" in our terminology, since the component is called "Dialog"?

Reply:
Ahh good call, I'm pretty inconsistent about this one...

Reply:
Looks like the `BackfillPreviewModal` is the only other modal, will make a little cleanup PR to rename that one too 👀

Reply:
https://github.com/dagster-io/dagster/pull/26930